logger/logger.go

```diff
@@ -65,7 +65,7 @@ func newEncoder(format string) (zapcore.Encoder, error) {
 	}
 }
 
-func newEncoderConfig() zapcore.EncoderConfig {
+func NewEncoderConfig() zapcore.EncoderConfig {
```

Comment:
I can't find any uses of `NewEncodingConfig` outside of the `logger` package. Is there a reason to change it to public?